_source/_posts/2018-12-20-crud-app-with-python-flask-react.md

```diff
@@ -1082,8 +1081,8 @@ class Home extends React.Component {
     kudos: []
   };
 
-  async componentDidMount() {
-    const accessToken = await this.props.authService.getAccessToken()
+  componentDidMount() {
+    const accessToken = this.props.authState.accessToken;
```

Comment:
This should be `this.props.authState.accessToken.accessToken`. See https://github.com/oktadeveloper/python-flask-react-crud-example/pull/8/commits/ce09dd9db96f0cee677f740441f04aed8ea853a0#diff-97cac7ad88dabf433a1eeb4a3fee7582d168a6bc1fdddfd4a9a8cdc0ece808e6R35. I created https://github.com/oktadeveloper/okta-blog/pull/690 to fix.